### PR TITLE
Ability to pass custom resolution/generalisation

### PR DESCRIPTION
**Pull request by edellucien**
# How to

Allow to pass data specific resolution array to LaterHelper function.
Ex: Lines geom in the search and mobz data have different resolutions available.

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

### CODE REVIEW

**Comment by edellucien** [comment]
Reset resolution/generalisation values to default.
Add ability to pass custom resolution/generalisation values.

File: src/layers/layerHelper.js [hunk 1]
```diff
@@ -1,18 +1,23 @@
-const dataResolutions = [900, 850, 500, 250, 115, 100, 75, 50, 20, 10, 5];
+const dataResolutions = [750, 500, 250, 100, 75, 50, 20, 10, 5];
 
-function getDataResolution(resolution) {
-  return dataResolutions.reduce((prev, curr) =>
+function getDataResolution(resolution, resolutions) {
```
Comment (oterral) [inline src/layers/layerHelper.js:3]:
```suggestion
function getDataResolution(resolution, resolutions = dataResolutions) {
```

File: src/layers/layerHelper.js [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+  return (resolutions || dataResolutions).reduce((prev, curr) =>
```
Comment (oterral) [inline src/layers/layerHelper.js:4]:
```suggestion
  return (resolutions.reduce((prev, curr) =>
```

**Comment by edellucien** [comment]
Thanks for the fast feedback!
I will apply the suggestions! Make sense to use parameter default value indeed!

**Pull request merged**